contributors/devel/README.md

```diff
@@ -49,10 +49,10 @@ Guide](http://kubernetes.io/docs/admin/).
 
 ## Developing against the Kubernetes API
 
-* The [REST API documentation](../api-reference/README.md) explains the REST
+* The [REST API documentation](http://kubernetes.io/docs/reference/) explains the REST
```

Comment:
Should we add some note how to get here: http://kubernetes.io/docs/api-reference/1_5/  - I don't want to hardcode the link, but a mention to how to get it would be awesome.  Also should we mention the swagger doc?